recipes/astrometry/post-link.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+FILE="$PREFIX/etc/astrometry.cfg"
+DEFAULT="$PREFIX/share/astrometry/astrometry.cfg"
+DATADIR="$PREFIX/data"
+
+# Avoid config file to be overwritten
+
+if [ -f $FILE ]; then
+    echo "Existing config file for astrometry.net."
```

Comment:
I think the documentation for post link scripts states they should never print output and should write to a specific file instead. Can you check?